build/v1/types.go

```diff
@@ -1349,6 +1349,13 @@ type SecretSpec struct {
 	MountPath string `json:"mountPath" protobuf:"bytes,2,opt,name=mountPath"`
 }
 
+const (
+	// BuildVolumeMountPath is where user defined BuildVolumes get mounted
+	BuildVolumeMountPath = "/var/run/openshift.io/volumes"
+	// BuildVolumeSuffix is a suffix for BuildVolume names
+	BuildVolumeSuffix = "user-build-volume"
```

Comment:
why do these belong in our public api?

Reply:
depending on how the API is guaranteed, this might be possible to change, but I suspect you'll want the mapping to remain consistent.